zos_basics/constructs/inventories/host_vars/zos_host.yml

```diff
@@ -1,23 +1,53 @@
 ---
 ################################################################################
-# Copyright (c) IBM Corporation 2021
+# Copyright (c) IBM Corporation 2021, 2024
 ################################################################################
 
 ################################################################################
 # Description of the properties used in this configuration:
 # - Property `PYZ` is the python installation home path on the z/OS managed node (target),
-#    e.g, pyz: "/usr/lpp/IBM/cyp/v3r8/pyz"
+#    e.g, pyz: "/usr/lpp/IBM/cyp/v3r12/pyz"
+# - Property `PYZ_VERSION` is the version of python on the z/OS managed node (target),
+#    e.g, pyz_version: "3.12"
 # - Property `ZOAU` is the ZOAU installation home on the z/OS managed node (target),
 #    e.g, zoau: "/usr/lpp/IBM/zoautil"
-# - property `ansible_python_interpreter` is the z/OS managed node (target) Python
+# - Property `ZOAU_PYTHONPATH` is the path to the ZOA Utilities python library.
+#    Specifically, the path points to the directory containing `zoautil_py`.
+#    If the wheel installation method is used with the `--target` or `--user` option,
+#   `ZOAU_PYTHONPATH` should point to the directory the wheel file was installed to.
+#    e.g ("/usr/zoau-wheel/")
+#    Else `ZOAU_PYTHONPATH` should point to the directory containing the pre-compiled binary.
+#    For ZOAU v1.2.x, the pre-compiled binary is located in the `lib` directory.
+#    e.g ("/usr/lpp/IBM/zoautil/lib" or "{{ ZOAU }}/lib")
+#    For ZOAU v1.3.x, the pre-compiled binary is located in a sub-directory
+#    corresponding to the `PYZ_VERSION` inside the `lib` directory,
+#    e.g ("/usr/lpp/IBM/zoautil/lib/3.12" or "{{ ZOAU }}/lib/{{ PYZ_VERSION }}")
+# - Property `ansible_python_interpreter` is the z/OS managed node (target) Python
 #    binary installation path, e.g, ansible_python_interpreter: "{{PYZ}}/bin/python3"
 #
-# Note, PYZ and ZOAU environment variables must be configured.
+# Note, PYZ, PYZ_VERSION, and ZOAU environment variables must be configured.
 ################################################################################
 
 PYZ: "path_to_python_installation_on_zos_target"
+PYZ_VERSION: "3.xx"
+
 ZOAU: "path_to_zoau_installation_on_zos_target"
 
+# Configuring the `ZOAU_PYTHONPATH` property is required.
+
+# If ZOAU installed by wheel with the --target or --user option, uncomment the following line:
+# ZOAU_PYTHONPATH: "{{ path_to_wheel_installation_directory }}"
+
+# If ZOAU is installed to the default location used by pip, the PYTHONPATH environment variable
+# does not need to be configured for the ZOAU python module to be found. Uncomment the following line:
+# ZOAU_PYTHONPATH: ""
```

Comment:
I think we might have discussed this but is it better to recommend an empty env var for this case or just state for them to not use the env var and remove it?

Reply:
It is stated a few lines higher up in that file that the environment variable must be defined (even if it's just an empty string). This is so that the all.yml file will have something to resolve when it goes to set "PYTHONPATH: {{ ZOAU_PYTHONPATH }}".

Better to have the user set the empty string here rather than instruct them to go touch another file (all.yml).

Another option would be to use a default variable definition in the other file, but that might unnecessarily complex-ify the other file

Reply:
Understood, makes sense, once the setuptools is dropped (hopefully) and soon precompiled binaries this will be simplified. I don't know if shell like defaults are supported, I suppose I can look up Jinja2 patterns, eg in shell `${host:=""}` reads as either `$host` or empty string as the default. 

I consider this resolved you explained the reasoning, thank you.